### ISSUE TITLE
Opening a folder while files are being dumped into that directory causes a crash

### ISSUE THREAD

**Issue by tdijital** [aerosoul94/FATXTools#4]
**Steps to reproduce**

1. Dump files into a folder
2. In windows file explorer open the parent folder that files are being dumped into

**Error log below**
```
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.IO.IOException: The process cannot access the file 'D:\xbox-dvt4(godfather)\DEVKIT' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.Directory.OpenHandle(String path)
   at System.IO.Directory.SetCreationTimeUtc(String path, DateTime creationTimeUtc)
   at System.IO.Directory.SetCreationTime(String path, DateTime creationTime)
   at FATX.Volume.DumpDirectory(String path, DirectoryEntry dirent) in C:\Users\xxx\Desktop\Backup Tools\fatx-tools-csharp\FATXTools\FATX\FileSystem\Volume.cs:line 293
   at FATX.Volume.DumpDirent(String path, DirectoryEntry dirent) in C:\Users\xxx\Desktop\Backup Tools\fatx-tools-csharp\FATXTools\FATX\FileSystem\Volume.cs:line 309
   at FATXTools.MainWindow.toolStripMenuItem3_Click(Object sender, EventArgs e) in C:\Users\xxx\Desktop\Backup Tools\fatx-tools-csharp\FATXTools\FATXTools\MainWindow.cs:line 414
   at System.Windows.Forms.ToolStripItem.RaiseEvent(Object key, EventArgs e)
   at System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   at System.Windows.Forms.ToolStripItem.FireEventInteractive(EventArgs e, ToolStripItemEventType met)
   at System.Windows.Forms.ToolStripItem.FireEvent(EventArgs e, ToolStripItemEventType met)
   at System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.ToolStripDropDown.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at System.Windows.Forms.ToolStripDropDown.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4121.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework/v4.0.30319/mscorlib.dll
----------------------------------------
FATXTools
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/Users/xxx/Desktop/Backup%20Tools/fatx-tools-csharp/FATXTools/FATXTools/bin/Debug/FATXTools.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4121.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4001.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3752.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3752.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4121.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3752.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
FATX
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/Users/xxx/Desktop/Backup%20Tools/fatx-tools-csharp/FATXTools/FATXTools/bin/Debug/FATX.DLL
----------------------------------------
netstandard
    Assembly Version: 2.0.0.0
    Win32 Version: 4.8.3752.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/netstandard/v4.0_2.0.0.0__cc7b13ffcd2ddd51/netstandard.dll
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.
```

**Comment by aerosoul94** [comment]
I added a dialog for file errors.
![image](https://user-images.githubusercontent.com/12718188/81630712-e94cf380-93cb-11ea-9145-e0589e065b52.png)

**Comment by JayFoxRox** [comment]
I think the dialog could be more user-friendly by removing the "Retry?" and instead making the buttons "Retry" and "Abort". I'm sure there's some other Windows dialogue you can steal this from.

**Comment by aerosoul94** [comment]
> I think the dialog could be more user-friendly by removing the "Retry?" and instead making the buttons "Retry" and "Abort". I'm sure there's some other Windows dialogue you can steal this from.
> 
> Also see `RetryCancel` at https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.messageboxbuttons?redirectedfrom=MSDN&view=netcore-3.1
> 
> ("Cancel" and "Try Again" are also popular choices for this)

I agree! Thanks for the suggestion. Fixed in 474677d.
